pkg/subctl/cmd/gather/resource.go

```diff
@@ -33,24 +33,48 @@ func ResourcesToYAMLFile(info *Info, ofType schema.GroupVersionResource, namespa
 			return errors.WithMessagef(err, "error listing %q", ofType.Resource)
 		}
 
-		path := filepath.Join(info.DirName, info.ClusterName+"-"+ofType.Resource+".yaml")
-		file, err := os.Create(path)
-		if err != nil {
-			return errors.WithMessagef(err, "error opening file %s", path)
+		if len(list.Items) == 0 {
+			info.Status.QueueWarningMessage(fmt.Sprintf("No %s found in namespace %s", ofType.Resource, namespace))
+			// also log to file
+			path := filepath.Join(info.DirName, "resources_not_found.log")
+			file, err := os.OpenFile(path, os.O_CREATE|os.O_APPEND|os.O_RDWR, 0666)
+			if err != nil {
+				return errors.WithMessagef(err, "error writing %s not found to %s", ofType.Resource, path)
+			}
+			defer file.Close()
+			_, err = file.WriteString(ofType.Resource)
+			if err != nil {
+				return errors.WithMessagef(err, "error writing to file %s", file)
+			}
+			return nil
```

Comment:
I think we should do this in a separate PR later if we decide and after we work out details (eg file name, what else we want log to it etc). I added a check list entry to https://github.com/submariner-io/submariner-operator/issues/1125.